src/components/Header/Header.tsx

```diff
@@ -0,0 +1,143 @@
+import {
+  useContext, useEffect, useMemo, useRef, useState,
+} from 'react';
+import classNames from 'classnames';
+import { TodosContext } from '../../contexts/TodosContext';
+import { USER_ID } from '../../variables';
+import { createTodo, updateTodo } from '../../api/todos';
+
+/* eslint-disable jsx-a11y/control-has-associated-label */
+export const Header = () => {
+  const {
+    title,
+    setTitle,
+    todos,
+    setTodos,
+    setErrorMessage,
+    setTempTodo,
+    tempTodo,
+    setChangedTodos,
+  } = useContext(TodosContext);
+
+  const [isDisabledButton, setIsDisabledButton] = useState(false);
+
+  const changeValues = () => {
+    setTempTodo({
```

Comment:
maybe think about better name for this function ` changeValues `

it looks like you add new value, create Todo